File: python-version/LC2643/row_and_maximum_ones.py
class Solution:
    def rowAndMaximumOnes(self, mat: list[list[int]]) -> list[int]:
        max_cnt_of_ones, index = -1, -1
        for i, row in enumerate(mat):
            ones_cnt = 0
            for m in row:
                ones_cnt += m
            if ones_cnt > max_cnt_of_ones:
                max_cnt_of_ones = ones_cnt
                index = i
        return [index, max_cnt_of_ones]

File: go-version/leetcode/LC2643/row_and_maximum_ones.go
package LC2643

/**
 * @Author: cutejiuge cutejiuge@163.com
 * @Date: 2025/3/22 下午1:01
 * @FilePath: row_and_maximum_ones
 * @Description: 遍历计数即可
 */

func rowAndMaximumOnes(mat [][]int) []int {
	maxCntOfOnes, index := -1, -1
	for i := 0; i < len(mat); i++ {
		onesCnt := 0
		for _, m := range mat[i] {
			onesCnt += m
		}
		if onesCnt > maxCntOfOnes {
			maxCntOfOnes = onesCnt
			index = i
		}
	}
	return []int{index, maxCntOfOnes}
}
